I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/daily_check.rpy", line 263, in script
    $ end_event('none', **kwargs)
  File "game/scripts/daily_check.rpy", line 263, in <module>
    $ end_event('none', **kwargs)
NameError: name 'kwargs' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/daily_check.rpy", line 263, in script
    $ end_event('none', **kwargs)
  File "C:\Users\bigma\Documents\RenPy\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\bigma\Documents\RenPy\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/scripts/daily_check.rpy", line 263, in <module>
    $ end_event('none', **kwargs)
NameError: name 'kwargs' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Mind the School 0.1.3
Wed Jan 24 16:22:47 2024
